configs/scnet/scnet_r50_fpn_20e_coco.py

```diff
@@ -0,0 +1,277 @@
+_base_ = [
+    '../_base_/datasets/coco_instance.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
```

Comment:
Is it possible to inherit configs from HTC of Cascade RCNN, to make config shorter?

Reply:
Thanks. This is a nice point.

Reply:
Btw, although inherit SCRoIHead from HTCRoIHead won't save much code. I still suggest doing so since they share similar components. Inheritance may be beneficial for future refactoring.